src/test/java/com/puppycrawl/tools/checkstyle/MainTest.java

```diff
@@ -302,14 +324,31 @@ public void testExistingTargetFileWithError()
             throws Exception {
         exit.expectSystemExitWithStatus(2);
         exit.checkAssertionAfterwards(() -> {
+            final LocalizedMessage auditStartMessage = new LocalizedMessage(0,
+                    Definitions.CHECKSTYLE_BUNDLE, "DefaultLogger.auditStarted",
+                    null, null, getClass(), null);
+            final LocalizedMessage auditFinishMessage = new LocalizedMessage(0,
+                    Definitions.CHECKSTYLE_BUNDLE, "DefaultLogger.auditFinished",
+                    null, null, getClass(), null);
```

Comment:
@romani Like I said below at https://github.com/checkstyle/checkstyle/pull/4553#discussion_r125308406 ,
These keys should be fields like we do for checks and their tests. I am fine with this being a public field in the `Main` class. This is similar and follows the pattern on how the checks are done.